src/almanack/metrics/metrics.yml

```diff
@@ -2,7 +2,7 @@
 metrics:
   - name: "repo-path"
     id: "SGA-META-0001"
-    result-type: "str"
+    result-type: "string"
```

Comment:
is there a difference between string and str?

Reply:
Thanks for catching this! I tried to correct and make more standard the use of `str` vs `string`. I'll make this correction.